linalg_ops/convolution/CMakeLists.txt

```diff
@@ -0,0 +1,149 @@
+# Copyright 2024 The IREE Authors
```

Comment:
The GPU and CPU tests seem to be following different formats, which makes the generated tests a bit harder to follow, and creates more files in `convolution/generated`. Can we use the same format for GPU and CPU, and reuse the same generated test files?